docs/sphinx/manual/building_hpx.rst

```diff
@@ -201,32 +201,29 @@ favorite compiler with |hpx| visit |hpx_buildbot|_.
    * * **Compilers**
      *
      *
-     *
    * * |gcc|_
-     * 7.0
+     * 7.5
```

Comment:
7.0 will most likely still work, modulo bugs etc. in gcc. I'd leave this as 7.0 even though we don't currently test it.